libs/components/core/src/lib/modules/overlay/overlay.service.ts

```diff
@@ -15,34 +16,18 @@ import { SkyOverlayComponent } from './overlay.component';
 
 /**
  * This service is used to create new overlays.
- * @dynamic
+ * @internal
```

Comment:
We don't export this service, but let's add `@internal` to make it obvious it's not part of the public API.